src/lib/Behat/PageElement/AdminList.php

```diff
@@ -43,25 +56,41 @@ public function clickTrashButton(): void
         $this->context->findElement($this->fields['trashButton'])->click();
     }
 
+    public function checkListElementSelectable(string $name): bool
+    {
+        $position = $this->context->getElementPositionByText($name, $this->fields['listElementLink']);
+        $checkbox = $this->context->findElement(sprintf($this->fields['tableCell'], $position, 1).' .form-check-input')->getAttribute("disabled");
+
+        return $checkbox !== 'disabled';
+    }
+
     public function selectListElement(string $name): void
     {
+        $position = $this->context->getElementPositionByText($name, $this->fields['listElementLink']);
+        $this->context->findElement(sprintf($this->fields['tableCell'], $position, 1))->checkField('');
     }
 
     public function clickListElement(string $name): void
     {
         $this->context->getElementByText($name, $this->fields['listElementLink'])->click();
     }
 
-    public function checkListElement(string $name): bool
+    public function checkListElementExists(string $name): bool
     {
         return $this->context->getElementByText($name, $this->fields['listElementLink']) !== null;
     }
 
-    public function verifyVisibility(): void
+    public function checkListItemAttribute(string $name, string $header): string
```

Comment:
This one returns texts (it does not check anything) - can't it be getTableCell($rowName, $columnName) / (getAdminTableCellValue)?.

To be honest, I don't like methods with "check".
We have:
`checkListItemAttribute` which does not check anything
`checkContentsStatus` which throws
`checkListElementExists`, `checkListElementSelectable` which do not throw and return bool

Maybe we could follow the following convention:
- if it throws: use `assert`/`verify` (`verifyAssignedContentCount`?) in the name?
- if it returns bool: sort-of-a-question: `isElementOnTheList`, `isElementSelectable`

What do you think?

Reply:
You're right, it might be confusing and it's a bad inconsistency. 
I'll fix it. Convention that you mention looks good for me.